### PR TITLE
[Feature] apply all in one chart in internal

### PR DESCRIPTION
**Pull request by Atralupus**
Implement all-in-one helm chart in internal cluster

### CODE REVIEW

File: 9c-internal/all-in-one/application.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
-  name: 9c-network
+  name: 9c-internal
```
Comment (sky1045) [inline 9c-internal/all-in-one/application.yaml:4]: out `9c-internal-v2` cluster is using `9c-network` namespace now. so if you dont want to change namespace, it shouldnt be updated

File: 9c-internal/all-in-one/application.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ spec:
   source:
     repoURL: https://github.com/planetarium/9c-infra.git
     targetRevision: main
-    path: 9c-internal/chart
+    path: ../../charts/all-in-one
```
Comment (sky1045) [inline 9c-internal/all-in-one/application.yaml:16]: should be from root path like `charts/all-in-one`

File: 9c-internal/all-in-one/application.yaml [hunk 3]
```diff
@@ -13,7 +13,7 @@ spec:
...
     helm:
       valueFiles:
         - values.yaml
```
Comment (sky1045) [inline 9c-internal/all-in-one/application.yaml:19]: and here reletive path from chart `../../9c-internal-v2/9c-network/values.yaml`

File: 9c-internal/all-in-one/values.yaml [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+# This is a YAML-formatted file.
+
+
+clusterName: "9c-internal-v2"
+logLevel: "debug"
+
+global:
+  image:
+    repository: planetariumhq/ninechronicles-headless
+    tag: "git-7c0815d08cb5d94700918ca4598ae2634c26ac1e"
```
Comment (sky1045) [inline 9c-internal/all-in-one/values.yaml:11]: as you can read from chart template, global image tag could be overwritten by the tag of each component(validator, remote-headless, etc.). so if you want to use global image tag, remove image tag from those using it

File: 9c-internal/all-in-one/values.yaml [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+seed:
+  count: 1
+  useTurnServer: false
+  image:
+    repository: planetariumhq/libplanet-seed
+    pullPolicy: Always # Overrides the image tag whose default is the chart appVersion.
+    tag: "git-22bed8ef41ab09258e6404462ec8e07cd038d089"
+
+  hosts:
+  - "9c-main-tcp-seed-1.planetarium.dev"
```
Comment (sky1045) [inline 9c-internal/all-in-one/values.yaml:45]: internal

File: 9c-internal/all-in-one/values.yaml [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+  storage:
+    data: 500Gi
+
+  resources:
+    requests:
+      cpu: 1
+      memory: 12Gi
+
+  nodeSelector:
+    beta.kubernetes.io/os: linux
```
Comment (sky1045) [inline 9c-internal/all-in-one/values.yaml:94]: i think we should checkout every nodeSelectors, validator also

**Review by sky1045 — CHANGES_REQUESTED** [review]

File: 9c-internal/all-in-one/values.yaml (hunk 6 above)
Comment (Atralupus) [reply 9c-internal/all-in-one/values.yaml:94]: Can you explain more?

Comment (sky1045) [reply 9c-internal/all-in-one/values.yaml:95]: https://github.com/planetarium/9c-infra/blob/main/9c-internal/chart/templates/remote-headless.yaml#L163-L164
as we can see above, we need more nodeSelector config for remote-headless, validator, etc.
previously these are injected directly to chart template, not by value

**Comment by Atralupus** [comment]
all done @sky1045

**Review by sky1045 — APPROVED** [review]
🚀

**Pull request merged**